engine/schema/src/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -104,210 +105,353 @@
 
 public class DatabaseUpgradeChecker implements SystemIntegrityChecker {
     private static final Logger s_logger = Logger.getLogger(DatabaseUpgradeChecker.class);
-
-    protected Map<CloudStackVersion, DbUpgrade[]> _upgradeMap = new HashMap<>();
-
-    @Inject
-    VersionDao _dao;
-
     private final ImmutableList<CloudStackVersion> availableVersions;
+    protected Map<CloudStackVersion, DbUpgrade[]> _upgradeMap = new HashMap<>();
+    @Inject VersionDao _dao;
 
     public DatabaseUpgradeChecker() {
```

Comment:
@DaanHoogland it will be hard to review the changes as there as all the newlines and formating has changed. Kindly revert some of that, for example having the `@Annotations` above the variable declarations.